client/src/pages/CreateProfilePage/CreateProfilePage.js

```diff
@@ -0,0 +1,169 @@
+  function handleDelete(tagIndex) {
+    const newTags = tags.filter((tag, i) => i != tagIndex);
+    setTags(newTags);
+  }
+
+  function toggleDone() {
+    //route to the filter view
+  }
+
+  function renderEdit() {
+    return (
+      <div className="create-profile-outer-div">
+        <div className="navbar">
+          <a href="google.com">
+            <img className="navbar-logo" src={logo} alt="cofed logo" />
+          </a>
+        </div>
+        <div className="create-profile-wrapper-div">
+          <div className="profile-top-div">
+            <Link to="/">
+              <button className="profile-done-button" onClick={toggleDone}>
+                Done
+              </button>
+            </Link>
+          </div>
+          <div className="profile-text-tags-container">
+            {renderContactInputs()}
+            <div className="profile-tags-container">
+              {tags &&
+                tags.map((text, index) => (
+                  <Tag
+                    key={index}
+                    text={text}
+                    index={index}
+                    onDelete={handleDelete}
+                  />
+                ))}
+            </div>
+          </div>
+          <div className="profile-descriptions">
+            <div className="create-profile-hr"></div>
+            <div className="profile-header">Mission Statement</div>
+            <textarea
+              className="profile-mission-input"
+              type="text"
+              placeholder="Enter mission statement:"
+              maxLength="350"
+              value={mission}
+              onChange={e => setMission(e.target.value)}
+            />
+            <div className="create-profile-hr"></div>
+            <div className="profile-header">Description</div>
+            <textarea
+              className="profile-description-input"
+              type="text"
+              placeholder="Enter description:"
+              maxLength="1000"
+              value={description}
+              onChange={e => setDescription(e.target.value)}
+            />
+            <div className="create-profile-hr"></div>
+          </div>
+          {renderEditSocials()}
+        </div>
+      </div>
+    );
+  }
+
+  function renderContactInputs() {
+    return (
+      <div className="profile-pic-text-container">
+        <div className="profile-pic-container">
+          <img className="profile-pic-edit" alt="Image" src={tina} />
+          <img className="profile-edit-pic" alt="Image" src={plusSign} />
+        </div>
+        <div className="profile-edit-profile-text-container">
+          <input
+            className="profile-name-input"
+            type="text"
+            placeholder="Enter name:"
+            value={name}
+            onChange={e => setName(e.target.value)}
+          />
+
+          <input
+            className="profile-small-input"
+            type="text"
+            placeholder="Enter location:"
+            value={location}
```

Comment:
the default value for `location` is displayed in the location input field on render. let's get rid of it :)

<img width="573" alt="Screen Shot 2020-10-30 at 11 15 04 PM" src="https://user-images.githubusercontent.com/20432204/97772580-c17e5300-1b05-11eb-8944-c843f22e7d69.png">